client/src/components/BulkCreateForm/MoreInformation/MoreInformation.js

```diff
@@ -1,258 +0,0 @@
-import React from 'react';
```

Comment:
Maybe it'll be useful (I guess we can ask Dr. Decker about this) to have some kind of explanation still about Bulk Upload is and how to use it?

Reply:
Yea I'm sure she'll want something. It was easier at the time to just wipe it away while I was working. We can come back to this PR and find the old info and work with it